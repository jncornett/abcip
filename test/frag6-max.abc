d ( stack="eth:ip6:frag6:udp" )
a ( frag6:max=1500; udp:len=2048 )
