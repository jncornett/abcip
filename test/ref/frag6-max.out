IP6 (flowlabel 0x00001, hlim 64, next-header Fragment (44) payload length: 1504) ::ffff:10.1.2.3 > ::ffff:10.9.8.7: frag (0x00000001:0|1496) 48620 > 8: UDP, length 2048
	0x0000:  0209 0807 0605 0201 0203 0405 86dd 6000  ..............`.
	0x0010:  0001 05e0 2c40 0000 0000 0000 0000 0000  ....,@..........
	0x0020:  ffff 0a01 0203 0000 0000 0000 0000 0000  ................
	0x0030:  ffff 0a09 0807 1100 0001 0000 0001 bdec  ................
	0x0040:  0008 0808 0904 4c6f 7265 6d20 6970 7375  ......Lorem.ipsu
	0x0050:  6d20 646f 6c6f 7220 7369 7420 616d 6574  m.dolor.sit.amet
	0x0060:  2c20 636f 6e73 6563 7465 7475 7220 6164  ,.consectetur.ad
	0x0070:  6970 6973 6369 6e67 2065 6c69 742e 2045  ipiscing.elit..E
	0x0080:  7469 616d 2073 7573 6369 7069 742c 2071  tiam.suscipit,.q
	0x0090:  7561 6d20 6174 2073 6365 6c65 7269 7371  uam.at.scelerisq
	0x00a0:  7565 2075 6c74 7269 6369 6573 2c20 6e69  ue.ultricies,.ni
	0x00b0:  7369 206e 756c 6c61 2066 6572 6d65 6e74  si.nulla.ferment
	0x00c0:  756d 206f 7263 692c 2061 2069 6d70 6572  um.orci,.a.imper
	0x00d0:  6469 6574 206c 6163 7573 2065 6e69 6d20  diet.lacus.enim.
	0x00e0:  6120 6665 6c69 732e 2043 756d 2073 6f63  a.felis..Cum.soc
	0x00f0:  6969 7320 6e61 746f 7175 6520 7065 6e61  iis.natoque.pena
	0x0100:  7469 6275 7320 6574 206d 6167 6e69 7320  tibus.et.magnis.
	0x0110:  6469 7320 7061 7274 7572 6965 6e74 206d  dis.parturient.m
	0x0120:  6f6e 7465 732c 206e 6173 6365 7475 7220  ontes,.nascetur.
	0x0130:  7269 6469 6375 6c75 7320 6d75 732e 204e  ridiculus.mus..N
	0x0140:  756c 6c61 6d20 6461 7069 6275 7320 7269  ullam.dapibus.ri
	0x0150:  7375 7320 696e 2064 6f6c 6f72 2069 6163  sus.in.dolor.iac
	0x0160:  756c 6973 2071 7569 7320 6c61 6369 6e69  ulis.quis.lacini
	0x0170:  6120 6573 7420 7065 6c6c 656e 7465 7371  a.est.pellentesq
	0x0180:  7565 2e20 4d61 6563 656e 6173 2075 7420  ue..Maecenas.ut.
	0x0190:  6665 6c69 7320 6573 742c 2073 6564 2069  felis.est,.sed.i
	0x01a0:  6e74 6572 6475 6d20 6c6f 7265 6d2e 2043  nterdum.lorem..C
	0x01b0:  7261 7320 6175 6374 6f72 2c20 6970 7375  ras.auctor,.ipsu
	0x01c0:  6d20 7669 7461 6520 7665 6e65 6e61 7469  m.vitae.venenati
	0x01d0:  7320 6661 7563 6962 7573 2c20 6e75 6e63  s.faucibus,.nunc
	0x01e0:  206d 6173 7361 2066 6572 6d65 6e74 756d  .massa.fermentum
	0x01f0:  2061 7567 7565 2c20 7669 7461 6520 756c  .augue,.vitae.ul
	0x0200:  6c61 6d63 6f72 7065 7220 7075 7275 7320  lamcorper.purus.
	0x0210:  6665 6c69 7320 7175 6973 2061 6e74 652e  felis.quis.ante.
	0x0220:  2046 7573 6365 2066 6575 6769 6174 2065  .Fusce.feugiat.e
	0x0230:  6e69 6d20 7574 206e 756e 6320 636f 6e73  nim.ut.nunc.cons
	0x0240:  6563 7465 7475 7220 6964 2075 6c74 7269  ectetur.id.ultri
	0x0250:  6369 6573 2074 6f72 746f 7220 6c6f 626f  cies.tortor.lobo
	0x0260:  7274 6973 2e20 4d61 6563 656e 6173 2069  rtis..Maecenas.i
	0x0270:  6e20 6f72 6369 206d 6574 7573 2c20 6e6f  n.orci.metus,.no
	0x0280:  6e20 696e 7465 7264 756d 206e 6571 7565  n.interdum.neque
	0x0290:  2e20 5065 6c6c 656e 7465 7371 7565 2076  ..Pellentesque.v
	0x02a0:  756c 7075 7461 7465 2061 6e74 6520 696e  ulputate.ante.in
	0x02b0:  2072 6973 7573 2075 6c6c 616d 636f 7270  .risus.ullamcorp
	0x02c0:  6572 2072 7574 7275 6d2e 2020 4675 7363  er.rutrum...Fusc
	0x02d0:  6520 7665 6c20 6461 7069 6275 7320 6c65  e.vel.dapibus.le
	0x02e0:  6f2e 2049 6e20 7365 6420 6c61 6375 7320  o..In.sed.lacus.
	0x02f0:  6e65 6320 6c61 6375 7320 7469 6e63 6964  nec.lacus.tincid
	0x0300:  756e 7420 696d 7065 7264 6965 7420 6120  unt.imperdiet.a.
	0x0310:  7665 6c20 6d61 676e 612e 2053 7573 7065  vel.magna..Suspe
	0x0320:  6e64 6973 7365 206c 7563 7475 7320 636f  ndisse.luctus.co
	0x0330:  6e67 7565 2069 6d70 6572 6469 6574 2e20  ngue.imperdiet..
	0x0340:  446f 6e65 6320 7365 6420 7361 6769 7474  Donec.sed.sagitt
	0x0350:  6973 206c 6f72 656d 2e20 496e 2068 6163  is.lorem..In.hac
	0x0360:  2068 6162 6974 6173 7365 2070 6c61 7465  .habitasse.plate
	0x0370:  6120 6469 6374 756d 7374 2e20 446f 6e65  a.dictumst..Done
	0x0380:  6320 736f 6c6c 6963 6974 7564 696e 2061  c.sollicitudin.a
	0x0390:  6e74 6520 6574 2061 6e74 6520 7469 6e63  nte.et.ante.tinc
	0x03a0:  6964 756e 7420 7574 2070 6f72 7461 2065  idunt.ut.porta.e
	0x03b0:  6c69 7420 7268 6f6e 6375 732e 2050 6861  lit.rhoncus..Pha
	0x03c0:  7365 6c6c 7573 2072 6973 7573 206e 756e  sellus.risus.nun
	0x03d0:  632c 2066 7269 6e67 696c 6c61 2061 2074  c,.fringilla.a.t
	0x03e0:  696e 6369 6475 6e74 2069 642c 206d 616c  incidunt.id,.mal
	0x03f0:  6573 7561 6461 2061 7420 7361 7069 656e  esuada.at.sapien
	0x0400:  2e20 4372 6173 2076 6974 6165 206d 6574  ..Cras.vitae.met
	0x0410:  7573 206a 7573 746f 2c20 6964 2061 7563  us.justo,.id.auc
	0x0420:  746f 7220 6d65 7475 732e 2043 7572 6162  tor.metus..Curab
	0x0430:  6974 7572 2073 7573 6369 7069 7420 6661  itur.suscipit.fa
	0x0440:  6369 6c69 7369 7320 6665 6c69 7320 7669  cilisis.felis.vi
	0x0450:  7461 6520 7068 6172 6574 7261 2e20 5574  tae.pharetra..Ut
	0x0460:  2065 7520 6e69 7369 206e 6973 692e 2043  .eu.nisi.nisi..C
	0x0470:  7261 7320 636f 6e64 696d 656e 7475 6d20  ras.condimentum.
	0x0480:  6575 6973 6d6f 6420 6d61 7373 6120 6575  euismod.massa.eu
	0x0490:  2074 7269 7374 6971 7565 2e20 446f 6e65  .tristique..Done
	0x04a0:  6320 6572 6f73 206e 6973 6c2c 2073 656d  c.eros.nisl,.sem
	0x04b0:  7065 7220 6574 206c 6163 696e 6961 2061  per.et.lacinia.a
	0x04c0:  2c20 7465 6d70 7573 2069 6420 6469 616d  ,.tempus.id.diam
	0x04d0:  2e20 4372 6173 206e 6f6e 2074 656c 6c75  ..Cras.non.tellu
	0x04e0:  7320 6e6f 6e20 7365 6d20 636f 6e67 7565  s.non.sem.congue
	0x04f0:  206d 6f6c 6c69 7320 6e6f 6e20 7574 206d  .mollis.non.ut.m
	0x0500:  692e 2044 7569 7320 6469 616d 2066 656c  i..Duis.diam.fel
	0x0510:  6973 2c20 6375 7273 7573 206e 6f6e 2073  is,.cursus.non.s
	0x0520:  7573 6369 7069 7420 6e6f 6e2c 206d 6f6c  uscipit.non,.mol
	0x0530:  6573 7469 6520 7669 7461 6520 7465 6c6c  estie.vitae.tell
	0x0540:  7573 2e20 446f 6e65 6320 6661 6369 6c69  us..Donec.facili
	0x0550:  7369 7320 6475 6920 6574 2065 726f 7320  sis.dui.et.eros.
	0x0560:  706f 7274 7469 746f 7220 636f 6e73 6571  porttitor.conseq
	0x0570:  7561 742e 2020 5574 2061 2062 6962 656e  uat...Ut.a.biben
	0x0580:  6475 6d20 6c69 6775 6c61 2e20 5072 6f69  dum.ligula..Proi
	0x0590:  6e20 6575 6973 6d6f 6420 706f 7274 6120  n.euismod.porta.
	0x05a0:  6c65 6f20 6120 6469 6374 756d 2e20 4165  leo.a.dictum..Ae
	0x05b0:  6e65 616e 2071 7569 7320 6970 7375 6d20  nean.quis.ipsum.
	0x05c0:  6d61 7572 6973 2c20 6575 2073 6167 6974  mauris,.eu.sagit
	0x05d0:  7469 7320 7361 7069 656e 2e20 4165 6e65  tis.sapien..Aene
	0x05e0:  616e 206e 6962 6820 6d65 7475 732c 206c  an.nibh.metus,.l
	0x05f0:  6f62 6f72 7469 7320 7365 6420 6665 7567  obortis.sed.feug
	0x0600:  6961 7420 7574 2c20 6665 726d 656e 7475  iat.ut,.fermentu
	0x0610:  6d20 7669 7461                           m.vita
IP6 (flowlabel 0x00001, hlim 64, next-header Fragment (44) payload length: 568) ::ffff:10.1.2.3 > ::ffff:10.9.8.7: frag (0x00000001:1496|560)
	0x0000:  0209 0807 0605 0201 0203 0405 86dd 6000  ..............`.
	0x0010:  0001 0238 2c40 0000 0000 0000 0000 0000  ...8,@..........
	0x0020:  ffff 0a01 0203 0000 0000 0000 0000 0000  ................
	0x0030:  ffff 0a09 0807 1100 05d8 0000 0001 6520  ..............e.
	0x0040:  6e69 6268 2e20 4372 6173 2073 6365 6c65  nibh..Cras.scele
	0x0050:  7269 7371 7565 2c20 6c69 6775 6c61 2065  risque,.ligula.e
	0x0060:  7420 616c 6971 7565 7420 7665 7374 6962  t.aliquet.vestib
	0x0070:  756c 756d 2c20 6572 6174 206d 6173 7361  ulum,.erat.massa
	0x0080:  2074 656d 7075 7320 6d69 2c20 6163 2070  .tempus.mi,.ac.p
	0x0090:  6f72 7474 6974 6f72 2074 7572 7069 7320  orttitor.turpis.
	0x00a0:  6d61 676e 6120 7665 6c20 7572 6e61 2e20  magna.vel.urna..
	0x00b0:  4e75 6e63 2069 6e20 6c69 6265 726f 2061  Nunc.in.libero.a
	0x00c0:  2061 7567 7565 2076 6573 7469 6275 6c75  .augue.vestibulu
	0x00d0:  6d20 6163 6375 6d73 616e 2061 7420 7574  m.accumsan.at.ut
	0x00e0:  2064 6961 6d2e 2043 7261 7320 6469 676e  .diam..Cras.dign
	0x00f0:  6973 7369 6d20 6661 6369 6c69 7369 7320  issim.facilisis.
	0x0100:  7269 7375 732c 2069 6e20 6469 6374 756d  risus,.in.dictum
	0x0110:  2066 656c 6973 2075 6c6c 616d 636f 7270  .felis.ullamcorp
	0x0120:  6572 2061 632e 2053 6564 2068 656e 6472  er.ac..Sed.hendr
	0x0130:  6572 6974 206c 6163 7573 2076 6974 6165  erit.lacus.vitae
	0x0140:  206d 6175 7269 7320 636f 6e73 6571 7561  .mauris.consequa
	0x0150:  7420 6e6f 6e20 6c61 6f72 6565 7420 6d65  t.non.laoreet.me
	0x0160:  7475 7320 636f 6e67 7565 2e20 4e75 6c6c  tus.congue..Null
	0x0170:  6120 6661 6369 6c69 7369 2e20 4e61 6d20  a.facilisi..Nam.
	0x0180:  7574 2064 6f6c 6f72 2065 6c69 742c 2065  ut.dolor.elit,.e
	0x0190:  7420 6772 6176 6964 6120 6e69 7369 2e20  t.gravida.nisi..
	0x01a0:  5068 6173 656c 6c75 7320 6961 6375 6c69  Phasellus.iaculi
	0x01b0:  7320 756c 7472 6963 6573 2070 6f72 7461  s.ultrices.porta
	0x01c0:  2e20 2043 756d 2073 6f63 6969 7320 6e61  ...Cum.sociis.na
	0x01d0:  746f 7175 6520 7065 6e61 7469 6275 7320  toque.penatibus.
	0x01e0:  6574 206d 6167 6e69 7320 6469 7320 7061  et.magnis.dis.pa
	0x01f0:  7274 7572 6965 6e74 206d 6f6e 7465 732c  rturient.montes,
	0x0200:  206e 6173 6365 7475 7220 7269 6469 6375  .nascetur.ridicu
	0x0210:  6c75 7320 6d75 732e 2041 656e 6561 6e20  lus.mus..Aenean.
	0x0220:  6175 6775 6520 7475 7270 6973 2c20 6d61  augue.turpis,.ma
	0x0230:  6c65 7375 6164 6120 6163 2076 6f6c 7574  lesuada.ac.volut
	0x0240:  7061 7420 6574 2c20 616c 6971 7561 6d20  pat.et,.aliquam.
	0x0250:  6120 6d61 7373 612e 2050 726f 696e 206e  a.massa..Proin.n
	0x0260:  6f6e 206d 6173 7361 206c 6962 6572       on.massa.liber
